queue_job/tests/common.py

```diff
@@ -227,13 +227,13 @@ def perform_enqueued_jobs(self):
         def by_graph(job):
             return job.graph_uuid or ""
 
-        sorted_jobs = sorted(self.enqueued_jobs, key=by_graph)
+        sorted_jobs = sorted(self.enqueued_jobs[:], key=by_graph)
```

Comment:
I don't believe the extra [:] slicing here is necessary.

Reply:
I agree, since `sorted` returns already a new list.